### DISCUSSION TITLE
Ender 3 v2 15x15 UBL configuration files

### DISCUSSION THREAD

**Discussion by Riley-N** [Jyers/Marlin#1747] [Q&A]
Hello I was wondering if anyone has or knows where to get the config files for the 15x15 UBL. I wanted to modify my firmware to higher temps and only saw the 10x10 config files available. Thanks!

**Comment by Cal-LF** [comment]
@Riley-N The config files would start with the 10x10 ubl and make needed changes then compile.

**Reply by Riley-N** [reply]
Okay this is what I thought I would have to do. So would I only have to change the GRID_MAX_POINTS_X from 10 to 15? There doesn't seem to be a value for the GRID_MAX_POINTS_Y just below it. Or is that value somewhere else? Sorry I am new to compiling and just downloaded VS Code yesterday. Thanks
![Screenshot 2022-01-02 6 05 22 PM](https://user-images.githubusercontent.com/96971016/147892806-ccf13651-a5b1-439d-ab46-e1d0985e2098.png)

**Reply by Cal-LF** [reply]
@Riley-N The line below states the Y value will be the same as X.
